### ISSUE TITLE
Replace deprecated pegdown markdown parser by flexmark-java

### ISSUE THREAD

**Issue by qwazer** [qwazer/markdown-confluence-gradle-plugin#13]
Replace deprecated [pegdown](https://github.com/sirthias/pegdown) markdown parser by [flexmark-java](https://github.com/vsch/flexmark-java)

Annoying issue is not solved in pegdown is broken nested lists: sirthias/pegdown/issues/245

**Comment by imanushin** [comment]
Will solution from https://github.com/md2conf/md2conf replace flexmark? 

E.g. am I right that potential desire can be use md2conf, which will help to move all code (except gradle configuration) to another project?

**Comment by qwazer** [comment]
Yes. I plan to deprecate this plugin in flavor of the `m2conf` toolset. The possible option is to create `md2conf-gradle-plugin` and `md2conf-maven-plugin` as wrappers over `m2conf` toolset. I didn't decide yet.
 The `md2conf` is still under developing. But core parts with `convert` and `publish` are almost ready.

BTW, If you interested I can give you maintenance rights to this project. I do not use `gradle` currently, and want to concentrate on `md2conf` project.

**Comment by imanushin** [comment]
Thank you. Now I'd like just to update dependencies and maybe get rid of Spring (however it is hard, because I will need to test my changes somehow), because of vulnerability checker (which complains about some dependencies).
